R/make_venn_diagram.R

```diff
@@ -0,0 +1,41 @@
+# make_venn_diagram.R
+
+# Purpose:  a helper function for clusterSystems and is unlikely to be useful elsewhere.
+
+# Parameters:
+# system_genes      vector of genes in the system
+# sys_name          name of the system
+# cluster_genes     vector of genes in the cluster
+# clust_num         Numeric identifier for the cluster
+# jaccard           The jaccard index of the set overlap of cluster and system
+# p                 P value of the Jaccar index
+#
+# Value: NULL
+
+# Author: Rachel Silverstein
+
+
+make_venn_diagram <- function(system_genes, sys_name, cluster_genes, clust_num, jaccard, p) {
+
+  clust_name <- paste(c("Cluster ", clust_num), collapse = "")
+
+
+  area1 <- length(system_genes)
+  area2 <- length(cluster_genes)
+  int <- length(intersect(system_genes, cluster_genes))
+  category <- c(sys_name, clust_name)
+
+  grid::grid.newpage()
```

Comment:
grid needs to be in `DESCRIPTION  depends:`

Reply:
Adding depends: grid produces a note:

Package in Depends field not imported from: ‘grid’
    These packages need to be imported from (in the NAMESPACE file)
    for when this namespace is loaded but not attached.

Not sure what this means?

Reply:
Sorry, I meant `imports:`
Explanation is here: https://stat.ethz.ch/pipermail/r-help/2013-October/361984.html and here https://stackoverflow.com/questions/8637993/better-explanation-of-when-to-use-imports-depends

I'll add it to imports and import it here.